### PR TITLE
Changed use files to one statement per line.

### PR DESCRIPTION
**Pull request by niklasborwell**
I suggest changing how we write statements in use files to this. 
Partly to make it more readable in general, and partly to avoid the unnecessary conflicts that arise when all the information is on the same row.

### CODE REVIEW

File: ooc-math.use [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
+Imports: Quaternion
+Imports: FloatComplex
+Imports: FloatRotation3D
+Imports: FloatVectorList
+Imports: FloatPoint2DVectorList
+Imports: FloatComplexVectorList
+Imports: FloatConvexHull2D
+Imports: BoolVectorList
+Imports: FloatRandomGenerator
+Imports: IntRandomGenerator
```
Comment (davidhesselbom) [inline ooc-math.use]: As per POSIX standards, all files need to end with a newline. This should be handled by your editor. Check your settings and amend.

File: ooc-base.use [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+Imports: Synchronized
+Imports: ReferenceCounter
+Imports: Event
+Imports: ThreadPool
+Imports: Container
+Imports: Timer
+Imports: Debug
+Imports: Profiler
+Imports: CLIParser
+Imports: StringBuilder
```
Comment (davidhesselbom) [inline ooc-base.use]: Would it be a good idea to sort this list alphabetically?

File: ooc-sdk-lite.use [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+Imports: lang/Exception
+Imports: lang/Format
+Imports: lang/IO
+Imports: lang/Iterators
+Imports: lang/Memory
+Imports: lang/Numbers
+Imports: lang/String
+Imports: lang/stdlib
+Imports: lang/types
+Imports: lang/VarArgs
```
Comment (davidhesselbom) [inline ooc-sdk-lite.use]: Hmm. If all of these classes are in `lang`, can't we change the `SourcePath` to `source/sdk/lang`?

File: ooc-math.use (hunk 1 above)
Comment (niklasborwell) [inline ooc-math.use]: Fixed.

File: ooc-sdk-lite.use (hunk 3 above)
Comment (niklasborwell) [inline ooc-sdk-lite.use]: I don't know..

File: ooc-base.use (hunk 2 above)
Comment (niklasborwell) [inline ooc-base.use]: I don't really think it matters.

Comment (davidhesselbom) [inline ooc-base.use]: Okay, so if someone adds another import here, where should it be added? At the bottom?

Comment (niklasborwell) [inline ooc-base.use]: I don't think it matters that much either. At the bottom, or rather at the end, is the way it was before.

File: ooc-sdk-lite.use (hunk 3 above)
Comment (simonmika) [inline ooc-sdk-lite.use]: We should investigate a change of source paths.

**Comment by davidhesselbom** [comment]
Looks good to me, aside from the questions above. @simonmika, should I merge this?

**Pull request merged**